interface/src/app/signature/enriched_genesets.tpl.html

```diff
@@ -12,13 +17,30 @@
     </tr>
   </thead>
 
-  <tr ng-repeat="geneset in enrichedGenesets">
+  <tr ng-repeat="geneset in enrichedGenesets | limitTo: numGenesetsShown">
```

Comment:
👍 I always forget about Angular's awesome filters. This is a great way to implement this feature.